samcli/commands/deploy/command.py

```diff
@@ -155,6 +155,12 @@
     help="Automatically resolve s3 bucket for non-guided deployments."
     "Do not use --s3-guided parameter with this option.",
 )
+@click.option(
+    "--acl",
+    default=None,
+    required=False,
+    help="Sets the specified canned ACL policy on uploaded artifact S3 objects."
+)
```

Comment:
Since this is shared between package and deploy have this option refactored as a shared option imported into both classes for best practice purposes! Thanks!